charts/caluma/templates/backend-deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: "{{ template "caluma.fullname" . }}-postgresql"
```

Comment:
Here we should go for a `existingSecret` value if there is one set, if not we should default to the subchart created secret.